examples/Console/TestJaegerExporter.cs

```diff
@@ -42,6 +44,25 @@ internal static object RunWithActivity(string host, int port)
                     {
                         o.AgentHost = host;
                         o.AgentPort = port;
+
+                        // Examples for the rest of the options, defaults unless otherwise specified
+                        o.MaxPayloadSizeInBytes = 4096;
+                        o.ProcessTags = new Dictionary<string, object>
```

Comment:
Sorry for the delay in getting to this PR.
I am not sure of the need for `ProcessTags` option. We can simply use Resource attributes for this purpose, and Jaeger will automatically pick all tags from the Resource. So exposing another option here is not needed?

Reply:
Discussed with @CodeBlanch and agreed that this is not required any longer as Resource API is the official recommendation for additional tags.

I'll submit a PR to remove it. This PR looks good, once the ProcessTags section is removed. Thank you!

Reply:
Sounds good @cijothomas , I just made the change to remove the process tags, appreciate it!